src/components/AsideSection/AsideSection.test.tsx

```diff
@@ -34,11 +35,23 @@ describe('<AsideSection />', () => {
         <AsideSection title="Test title">
           <div>Child Content</div>
         </AsideSection>,
-      );
+      ).dive();
     });
 
     it('matches the snapshot', () => {
       expect(component).toMatchSnapshot();
     });
   });
+
+  describe('with customizer', () => {
+    it('receives custom theme', () => {
+      const theme = { palette: {}, semanticColors: {} };
+      const mountedComponent = mount(
+        <Customizer settings={{ theme }}>
+          <AsideSection title="TITLE" />
+        </Customizer>,
+      );
+      expect(mountedComponent.find('AsideSection').prop('theme')).toBe(theme);
+    });
+  });
```

Comment:
I'm planning to repeat this test for every `customizable` component.

However it's not yet entirely clear to me why I wasn't able to pull the theme prop using a `shallow` render with `dive`. Is `mount` fine here?